platform/security/sts/security-sts-propertyclaimshandler/src/main/java/org/codice/ddf/security/sts/claims/property/AttributeFileClaimsHandler.java

```diff
@@ -148,16 +133,50 @@ public ProcessedClaimCollection retrieveClaimValues(
   }
 
   @Override
-  public List<String> getSupportedRealms() {
-    return supportedRealms;
+  public Optional<List<Serializable>> getValues(final String attributeName) {
+    return Optional.ofNullable(systemHighAttributes.get(attributeName));
```

Comment:
Should probably pass back an unmodifiable list.

Reply:
```java
        Collections.unmodifiableList(systemHighAttributes.get(attributeName)));
```
will throw a NPE when `systemHighAttributes` does not contain key `attributeName`.

To keep this method small, I added updated Line 179 instead:
```java
        newSystemHighAttributes.put(
            systemHighAttributeName,
            Collections.unmodifiableList((List<Serializable>) systemHighAttribute.getValue()));
```